sql/server.go

```diff
@@ -139,7 +198,10 @@ func (s *Server) exec(req driver.Request) (driver.Response, error) {
 		return resp, err
 	}
 	for _, stmt := range stmts {
-		// TODO(pmattis): Fill placeholders.
+		// Bind all the placeholder variables in the stmt to actual values.
+		if err = s.bindValues(stmt, parameters{params: req.Params}); err != nil {
```

Comment:
I think binding the values here is correct. `parser.WalkExpr` is going to always walk the full expression tree. Rather than `Server.bindValues`, which doesn't actually need anything in `Server`, I think you should add `parser.WalkStmt` so that all of the binding occurs in the `parser` package.